docs/resources/database.md

```diff
@@ -1 +1,464 @@
-# :resources-database: Database
\ No newline at end of file
+# :resources-database: Database
+
+Database as a [Resource](/resources/) within DataOS is declared for creating a database within DataOS. It is treated as a first-class resource, encapsulating the fundamental unit for storing and organizing structured data using PostgreSQL. The classification of Databases as a resource type within the DataOS ecosystem allows for systematic management, provisioning, and utilization of databases in a standardized manner.
+
```

Comment:
A Database Resource in DataOS act as a repository for storing transaction data, utilizing a managed Postgres relational database. It fulfills the transactional data storage needs of custom data applications developed on top of DataOS. Internal applications like Lens, Metis, etc., also leverage the Database Resource for their transactional data storage. Utilizing a [Service](/resources/service/) supported by the [Beacon Stack](/resources/stacks/beacon/) facilitates CRUD operations (Create, Read, Update, Delete) on data assets stored in the Database, ensuring smooth connectivity between the data application and the Database. For analytical operations, users can utilize the [Flare Stack](/resources/stacks/flare/) to move the data from transactional data storage to other [Depots](/resources/depot/) or [Lakehouses](/resources/lakehouse/).